doc/releases/release-notes-3.2.rst

```diff
@@ -175,6 +175,14 @@ Architectures
 
 * Xtensa
 
+  * Macros :c:macro:`RSR` and :c:macro:`WSR` have been renamed to
```

Comment:
don't use `:c:macro:` for deleted macros, use literals instead

Reply:
removed.